content/en/blog/_posts/2024-12-11-Kubernetes-v1-32-Release.md

```diff
@@ -0,0 +1,302 @@
+
+### Support to size memory backed volumes
+
+The feature makes it possible to dynamically size memory-backed volumes based on pod resource limits. This improves the workload's portability and overall node resource utilization for the users.
+    
+This work was done as a part of [KEP #1967](https://github.com/kubernetes/enhancements/issues/1967), by [SIG Node](https://github.com/kubernetes/community/tree/master/sig-node).
+
+### Job API managed-by mechanism
+
+Support of the `managedBy` field has been added for Jobs and it is stable in 1.32 release. This feature enables external controllers (like [Kueue](https://kueue.sigs.k8s.io/)) to manage Job synchronization, offering greater flexibility and integration with advanced workload management systems.
+
+This work was done as a part of [KEP #4368](https://github.com/kubernetes/enhancements/issues/4368), by [SIG Apps](https://github.com/kubernetes/community/tree/master/sig-apps).
+
+## Features graduating to Beta
+
+_This is a selection of some of the improvements that are now beta following the v1.32 release._
+
+### Only allow anonymous auth for configured endpoints
+
+Allow users to specify which endpoints are allowed for anonymous requests. This allows the admin to only allow access to health endpoints like `/healthz`, `/livez` and `/readyz` anonymously while making sure other cluster endpoints or resources cannot be access anonymously even if a user misconfigures RBAC.
+    
+This work was done as a part of [KEP #4633](https://github.com/kubernetes/enhancements/issues/4633), by [SIG Auth](https://github.com/kubernetes/community/tree/master/sig-auth).
+
+### Per-plugin callback functions for accurate requeueing in kube-scheduler enhancements
+
+Enhance the scheduling throughput with more efficient scheduling retry decisions by per-plugin callback functions (QueueingHint). 
+We've implemented QueueingHints in all plugins, and are moving ahead enabling this feature by default.
+
+This work was done as a part of [KEP #4247](https://github.com/kubernetes/enhancements/issues/4247), by [SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling).
+
+### Recover from volume expansion failure
+
+This feature moved to Beta in v1.32, allowing users to recover from volume expansion failure by retrying with a smaller size. This enhancement ensures that volume expansion is more resilient and reliable, reducing the risk of data loss or corruption during the process.
+
+This work was done as a part of [KEP #1790](https://github.com/kubernetes/enhancements/issues/1790), by [SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+
+### Volume group snapshot
+
+This feature introduces a VolumeGroupSnapshot API to take a snapshot of multiple volumes together.
+Allows users to take a snapshot of multiple volumes together, ensuring data consistency across the volumes.
```

Comment:
```suggestion
This feature introduces a VolumeGroupSnapshot API to take a snapshot of multiple volumes together to
ensure data consistency across the volumes.
```

First half of second sentence is a repeat of first sentence

Reply:
Whoops, fixed.